saleor/graphql/product/tests/queries/test_product_query.py

```diff
@@ -12,6 +12,7 @@
 from .....attribute.utils import associate_attribute_values_to_instance
 from .....core.units import WeightUnits
 from .....product.models import (
+    Channel,
```

Comment:
This import is no longer used, right?

Reply:
I will continue one I get off work later today.